packages/sessions-sdk/src/index.ts

```diff
@@ -100,6 +100,24 @@ const createSession = async (
     adapter.sendTransaction(sessionKey, instructions),
 });
 
+enum SymbolOrMintType {
+  Symbol = "Symbol",
+  Mint = "Mint",
+}
+
+const SymbolOrMint = {
+  Symbol: (symbol: string) =>
+    ({
+      type: SymbolOrMintType.Symbol,
+      symbol,
+    }) as const,
```

Comment:
Minor nit but it's better to declare `as const` on the smallest scope needed since you usually don't want types to be narrowed more than they usually are, so in this case you'd do `Symbol: (symbol: string) => ({ type: SymbolOrMintType.Symbol as const, symbol })`